db/model.js

```diff
@@ -0,0 +1,33 @@
+const mongoose = require("mongoose");
+
+const commentSchema = new mongoose.Schema({
+  user_id: {
+    type: Number,
+    required: true,
+  },
+  song_id: {
+    type: Number,
+    required: true,
+  },
+  content: {
+    type: String,
+    required: true,
+  },
+  time_stamp: {
+    type: Number,
+    required: true,
```

Comment:
There is a date function for timestamps. Are you sure you should use a number here?

Reply:
Hmm, I didn't change it from the inherited data schema. Dates are a little bit different from a second count I think (which is what's happening here.)